sailthru/sailthru_content/services/catalog_api_service.py

```diff
@@ -63,25 +63,18 @@ def get_courses(self):
         logger.debug('Get Courses called')
         return self._get_resource_from_api(self.api_client.courses(), COURSES_PAGE_SIZE, marketable=1)
 
-    def get_searchable_course_keys(self):
+    def get_searchable_course_run_keys(self):
         logger.debug('Get Searchable Courses called')
         searchable_course_keys = []
-        searchable_courses = self._get_resource_from_search_api(self.api_client.search.all.facets())
-        for course in searchable_courses:
-            if course.get('key') and course.get('content_type') != 'program':
-                searchable_course_keys.append(course['key'])
+        searchable_course_runs = self._get_resource_from_search_api(
+            self.api_client.search.all.facets(), content_type='courserun'
```

Comment:
Thank you for changing this to provide the content_type.  

I understand why we make the call to the search endpoint, because we want only the marketable courses.  I don't understand why you make the other call to the courses endpoint to get the course_run data ( courses = get_courses()).  Don't you have all of the information you need from the search endpoint?  I took a quick look at the fields and they all seem to be there.  Can you explain to me why we need to make two separate calls here?